backend/src/handlers/routes.py

```diff
@@ -71,6 +77,73 @@ def get_routes(
         return routes_json
 
 
+@router.get("/kmlfile")
+def get_kml(req: Request):
+    """
+    Gets the KML file for all routes.
+    """
+
+    print("test")
+
+    with req.app.state.db.session() as session:
+        routes = session.query(Route).all()
+
+        stops = session.query(Stop).all()
+
+        route_stops = session.query(RouteStop).all()
+
+        k = kml.KML()
+        ns = "{http://www.opengis.net/kml/2.2}"
+        d = kml.Document(ns, "3.14", "Routes", "Routes for the OreCart app.")
+        k.append(d)
+
+        style = kml.Style(id="route-outline")
+        style.append_style(
+            LineStyle(color="ff0000ff", width=2)
+        )  # Red outline in AABBGGRR hex format
+        style.append_style(PolyStyle(fill=0))  # No fill
+
+        for route in routes:
+            p = kml.Placemark(ns, route.name, route.name, route.name)
+            p.geometry = Polygon([(w.lon, w.lat, 0) for w in route.waypoints])
```

Comment:
It'll be best in the future to add a position field IMO just to ensure this is always ordered right

Reply:
This hasn't proven to be a problem because of the way that waypoints are formatted in kml and added to the database. So idc about this for now.